packages/core/src/telemetry/service.ts

```diff
@@ -78,12 +160,46 @@ export class TelemetryService {
     return hash.digest("hex");
   }
 
-  private async getProjectId() {
-    if (this.rawProjectId) {
-      return this.oneWayHash(this.rawProjectId);
+  private flushDetached() {
+    const serializedEvents = JSON.stringify(this.events);
+    const fileName = `${this.options.ponderDir}/telemetry-events.json`;
```

Comment:
I think you should probably `path.join()` these (pretty sure this code would break on Windows).